sections/appendix/stemming.tex

```diff
@@ -1,24 +1,29 @@
 \subsection{Stemming the dataset}\label{sec:stemming}
-\vejleder{Se dokument for flere kommentarer}
-In this project, we have done minimal preprocessing \vejleder{hvad med stop ord, fylde ord?}of the dataset, because, in a previous project, a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
+In this project, we have done minimal preprocessing of the dataset.
+Stop words have not been specifically removed, but most of these are naturally removed since we filter all words that appear in more than $10\%$ of the documents.
+We do minimal preprocessing, because in a previous project a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
```

Comment:
```suggestion
We do minimal preprocessing because in a previous project a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
```